qmail-rspawn.c

```diff
@@ -83,14 +84,24 @@ int len;
     }
 }
 
+static char *setup_qrargs()
+{
+ static char *qr;
```

Comment:
is it not required to set `qr = NULL` here ?
I fear `if (qr)` just below has undefined behavior as on the first run, `qr` has no value set.

Using a static variable for caching the result of env_get sounds good though.

Reply:
static variables are 0 by default.